src/NzbDrone.Core/Notifications/Pushsafer/PushsaferService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Extensions;
+using RestSharp;
+using NzbDrone.Core.Rest;
+
+namespace NzbDrone.Core.Notifications.Pushsafer
+{
+    public interface IPushsaferProxy
+    {
+        void SendNotification(string title, string message, PushsaferSettings settings);
+        ValidationFailure Test(PushsaferSettings settings);
+    }
+
+    public class PushsaferProxy : IPushsaferProxy
+    {
+        private readonly Logger _logger;
+        private const string URL = "https://www.pushsafer.com/api";
+
+        public PushsaferProxy(Logger logger)
+        {
+            _logger = logger;
+        }
+
+        public void SendNotification(string title, string message, PushsaferSettings settings)
+        {
+            var client = RestClientFactory.BuildClient(URL);
+            var request = new RestRequest(Method.POST);
+            request.AddParameter("k", settings.ApiKey);
+            request.AddParameter("t", title);
+            request.AddParameter("m", message);
+
+            if (!settings.Sound.IsNullOrWhiteSpace())
```

Comment:
`.IsNotNullOrWhiteSpace()` eliminates the negation. (twice)

Reply:
Will replace. thanks

Reply:
This is the same approach that is used for a different notification: Pushover. I dont know if its worth changing there too or not.

Reply:
IIRC `IsNotNullOrWhiteSpace` was added later, would be good to fix those up as well as a separate commit.